stylesheets/manifest.css

```diff
@@ -1525,6 +1525,8 @@ li.entry .error-icon-container {
       background-color: silver;
       height: 100%;
       width: 100%; }
+  	.discussion-container .scroll-down-button-view button .icon:hover {
+    		background-color: #616161; }
```

Comment:
This project uses sass to "compile" stylesheets. You have to edit the sass (.scss) files and run grunt to compile instead of this plain css file.
Rel: https://github.com/WhisperSystems/Signal-Desktop/pull/1176#pullrequestreview-39344890